MAUI/Switch/customization.md

```diff
@@ -229,7 +229,10 @@ this.Content = sfSwitch;
 
 ## Custom Path
 
-In the switch control, you can customize the thumb's icon by providing a custom path, the custom path should be given to the `Path` property.
+In the switch control, you can customize the thumb's icon by using the following properties:
```

Comment:
In the switch control, customize the thumb's icon by using the following properties: